<launch>
	<node  name="og_labeler" pkg="era_gazebo" type="og_labeler" output="screen">
	
	<param name="use_sim_time" value="true" />

	<remap from="~/object_input" to="/detected_objects" />
	<remap from="~/image_input" to="/camera/aligned_depth_to_color/image_raw" />
	<remap from="~/camera_info" to="/camera/aligned_depth_to_color/camera_info" />


	</node>
</launch>